mmseg/models/builder.py

```diff
@@ -46,3 +49,40 @@ def build_segmentor(cfg, train_cfg=None, test_cfg=None):
         'test_cfg specified in both outer field and model field '
     return SEGMENTORS.build(
         cfg, default_args=dict(train_cfg=train_cfg, test_cfg=test_cfg))
+
+
+def build_upsample_layer(cfg, *args, **kwargs):
```

Comment:
Shall we use `UPSAMPLER_LAYERS.build()`?

Reply:
https://github.com/open-mmlab/mmcv/blob/c47c9196d067a0900b7b8987a8e82768edab2fff/mmcv/cnn/bricks/upsample.py#L50

```python
def build_upsample_layer(cfg, *args, **kwargs):
    UPSAMPLER_LAYERS.build(cfg, *args, **kwargs)
```
will miss some checks (L72-81, L85-86).

Reply:
In addition, if we use `build_upsample_layer` from MMCV, we need to add `mmseg` scope for `type` to build the corresponding upsample layer.

Reply:
> Shall we use `UPSAMPLER_LAYERS.build()`?

Updated.